packages/host/tests/integration/components/ai-assistant-panel/codeblocks-test.gts

```diff
@@ -637,4 +637,45 @@ Above code blocks are now complete`;
 
     await percySnapshot(assert);
   });
+
+  test('it will render diff editor for a blank file', async function (assert) {
+    let roomId = await renderAiAssistantPanel(`${testRealmURL}Person/fadhlan`);
+    let messageWithSearchAndReplaceBlock = `Here's some HTML inside codeblock with search and replace block:
+
+\`\`\`txt
+// File url: https://example.com/blank.txt
+<<<<<<< SEARCH
+
+=======
```

Comment:
Does it also work without the blank line in between SEARCH and ======? Does it need to?

Reply:
I’ve removed the blank line from the test, when I was working on this locally, I had one because of editor autosave formatting, but it’s not needed, thanks.